### PR TITLE
feat: add functional menu canvas and buttons

### PR DESCRIPTION
**Pull request by SSelinn**
## Description

In the scene Menu, I have added a menu and settings canvas along with buttons and their functionality.

## Testability

You can press the settings button and go back (contents is in another castle), than you can press the play button and you will be transported to the melee scene.

Note: It is true that the quit button doesn't "work". You need to make a build of unity, open the aplication and than press the quit button.

## Checklist
- [x] Set the proper pull request name
- [x] Merged main into your branch
- [x] Added a scene to the game for your changes

### CODE REVIEW

File: aplib.net-demo/Assets/Scripts/MainMenu/MenuButtons.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    /// <summary>
+    /// When the play button in the main menu is clicked, it wil teleport the player to the game.
+    /// Right now it is the melee weapon scene to prove concept. THIS NEEDS TO BE CHANGED.
+    /// </summary>
+    public void PlayGame()
+    {
+        SceneManager.LoadScene("MeleeWeapon");
+    }
```
Comment (JoenvandeVorle) [inline aplib.net-demo/Assets/Scripts/MainMenu/MenuButtons.cs:35]:
```suggestion
    /// <summary>
    /// When the play button in the main menu is clicked, it wil teleport the player to the game.
    /// Right now it is the melee weapon scene to prove concept.
    /// </summary>
    public void PlayGame()
    {
        // TODO:: Load main game scene
        SceneManager.LoadScene("MeleeWeapon");
    }
```

**Review by JoenvandeVorle — CHANGES_REQUESTED** [review]

File: aplib.net-demo/Assets/Scripts/MainMenu/MenuButtons.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        {
+            settingCanvas.SetActive(false);
+            isOnSettings = false;
+            menuCanvas.SetActive(true);
+        }
+        else
+        {
+            menuCanvas.SetActive(false);
+            isOnSettings = true;
+            settingCanvas.SetActive(true);
```
Comment (Tboefijn) [inline aplib.net-demo/Assets/Scripts/MainMenu/MenuButtons.cs:52]: Ik weet niet of hier later nog meer achterzit maar je kan het omschrijven hierheen.
```suggestion
        settingCanvas.SetActive(false);
        isOnSettings = !isOnSettings;
        menuCanvas.SetActive(true);
```

**Review by JoenvandeVorle — CHANGES_REQUESTED** [review]
left a small comment

**Review by Tboefijn — CHANGES_REQUESTED** [review]
Looks good! I left one comment.

File: aplib.net-demo/Assets/Scripts/MainMenu/MenuButtons.cs (hunk 2 above)
Comment (SSelinn) [reply aplib.net-demo/Assets/Scripts/MainMenu/MenuButtons.cs:53]: Voor de isOnSetting is het idd handig, alleen menu en setting togglen ook. Ik heb misschien een idee hiervoor, let me cook.

**Review by JoenvandeVorle — APPROVED** [review]
lgtm

**Review by Tboefijn — APPROVED** [review]
LGTM!

**Pull request merged**